README.md

```diff
@@ -83,6 +83,10 @@ For [code execution](https://microsoft.github.io/autogen/docs/FAQ/#code-executio
 
 For LLM inference configurations, check the [FAQs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints).
 
+## Simple Example in ChatGPT Style
+
+One [simple example](https://github.com/microsoft/autogen/blob/main/samples/simple_chat.py) of human chatting with an AutoGen agent in ChatGPT style.
```

Comment:
For clarity, this is not an example of a human chatting, as one might find in a notebook. Instead, this is sample code allowing the user to chat, etc. I word reword the line this way:

"Sample code allowing the user to chat with an AutoGen agent in ChatGPT style."

Reply:
done